src/worldcereal/openeo/postprocess.py

```diff
@@ -0,0 +1,153 @@
+import xarray as xr
+from openeo_gfmap.inference.model_inference import ModelInference
+
+
+class PostProcessor(ModelInference):
+    """Perform post-processing from the model outputs. Expects an input cube
+    with 2 + N bands, where N is the number of classes. The first band is the
+    prediction value, the second class is the max probability and the rest are
+    the per-class probabilities.
+
+    Interesting UDF parameters:
+    is_binary: bool
+        If the postprocessing is applied on a binary classification model (cropland)
+        or a multi-class classification model (croptype). Default is False.
+    lookup_table: Optional[dict]
+        Required if is_binary is False. A lookup table to map the class names
+        to class labels, ordered by model output.
+    """
+
+    EXCLUDED_VALUES = [254, 255, 65535]
+
+    def output_labels(self) -> list:
+        return ["classification", "max_probability"]
```

Comment:
see earlier comment, we should probably call the final result just "probability". Besides, we may want to build in the option to still save all probabilities to the result in case we want crop probability maps.

Reply:
I can add that option, do you want to keep the old probabilities (before postprocessing) or the ones after?

Reply:
The ones after I guess, otherwise we can use the intermediate saving on the classification, not?

Reply:
This was added